scipy/stats/_stats_py.py

```diff
@@ -6190,6 +6199,11 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     recommended if the underlying distribution is long-tailed or contaminated
     with outliers [4]_.
 
+    The statistic is calculated as ``(np.mean(a) - np.mean(b))/se``, where
+    ``se`` is the standard error. Therefore, the statistic will be positive
+    when the sample mean of `a` is greater than the sample mean of `b` and
+    negative when the sample mean of `a` is less than the sample mean of
+    `b`.
```

Comment:
I think this will fix:
```/home/circleci/repo/build/testenv/lib/python3.8/site-packages/scipy/stats/_stats_py.py:docstring of scipy.stats._stats_py.ttest_ind:19: WARNING: citation not found: R3566833beaa2-1```

```suggestion
    `b`.

```